core/src/main/java/com/linecorp/armeria/client/proxy/ProxyConfig.java

```diff
@@ -110,15 +108,10 @@ public static ConnectProxyConfig connect(
     }
 
     /**
-     * Returns a {@code ProxyConfig} which signifies that proxy is disabled.
+     * Returns a {@code ProxyConfig} which signifies that a proxy is absent.
      */
-    public static ProxyConfig disabled() {
-        return DISABLED_PROXY_CONFIG;
-    }
-
-    @Nullable
-    static String maskedStr(@Nullable String value) {
-        return value == null ? null : "****";
```

Comment:
How about reviving this and using `omitNullValues()` for `ToStringHelper`? Showing `****` for a non-existent password may give a user a misconception that authentication is enabled.

Reply:
We could still return `****` when `username` is not `null`, though.